### PR TITLE
Batch database queries using bulk_create_manager.py

### PR DESCRIPTION
**Pull request by gernathlub**
bulk update and delete in predict_depths method

### CODE REVIEW

**Comment by gernathlub** [comment]
fixes #92

File: manyfews/calculations/flood_risk.py [hunk 1]
```diff
@@ -105,63 +106,58 @@ def predict_depths(forecast_time, param_ids, flow_values):
...
+                    date=forecast_time,
+                    parameters_id=param.id,
+                    model_version=param.model_version,
+                    median_depth=median,
+                    lower_centile=lower_centile,
+                    mid_lower_centile=mid_lower_centile,
+                    upper_centile=upper_centile,
+                )
+                if prediction:  # update:
+                    new_prediction.pk = prediction.pk
```
Comment (sjmf) [inline manyfews/calculations/flood_risk.py:151]: Ah excellent! I really like this way of doing it.

File: manyfews/calculations/tests.py [hunk 2]
```diff
@@ -325,3 +347,40 @@ def test_predict_depth(self):
...
+        predict_depth.return_value = (8.14, 21.95, 36.63, 1)
+        self.create_depth_predictions()
+
+        dummy_param_list = [1, 2, 3, 4]
+
+        self.assertEqual(DepthPrediction.objects.filter(date=self.test_date).count(), 4)
+        with self.assertNumQueries(10):
+            predict_depths(self.test_date, dummy_param_list, None)
+
+        self.assertEqual(DepthPrediction.objects.filter(date=self.test_date).count(), 4)
```
Comment (sjmf) [inline manyfews/calculations/tests.py:386]: Brilliant, thank you!

**Review by sjmf — APPROVED** [review]

**Comment by sjmf** [comment]
Thanks again for all your work @gernathlub . We really appreciate it!

**Pull request merged**